### PR TITLE
Admin user index page

### PR DESCRIPTION
**Pull request by grwthomps**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,10 +3,14 @@ class User < ApplicationRecord
...
+  belongs_to :merchant, optional: true
 
   enum role: %w(default merchant_employee merchant_admin admin)
 
   validates_numericality_of :zip, only_integer: true
   validates_length_of :zip, is: 5
+
+  def titleize_role
+    role.titleize
+  end
```
Comment (johnktravers) [inline app/models/user.rb:15]: Nice helper method. Very useful

File: app/views/admin/users/index.html.erb [hunk 2]
```diff
@@ -1 +1,20 @@
...
+  <tbody>
+    <% @users.each do |user| %>
+      <tr id="user-<%= user.id %>">
+        <th scope="row"><%= link_to "#{user.name}", "/admin/users/#{user.id}" %></th>
+        <td><%= user.created_at.strftime("%m/%d/%Y") %></td>
+        <td><%= user.titleize_role %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```
Comment (johnktravers) [inline app/views/admin/users/index.html.erb:20]: Table looks great!

File: app/models/user.rb (hunk 1 above)
Comment (zacisaacson) [inline app/models/user.rb:15]: titleize. nice!

**Review by zacisaacson — APPROVED** [review]

**Review by johnktravers — COMMENTED** [review]
Looks great, good work!

**Pull request merged**